src/utils/misc.ts

```diff
@@ -38,3 +40,15 @@ export const openInWebBrowser = (url: string) => {
     }
   }
 }
+
+export const stringToDoubleSHA215HexString = (data: string): string => {
```

Comment:
I would suggest testing this with some test vectors. Crypto functions are hard to use sometimes...

Reply:
Yeah, we should have some tests for sure...!